src/util.js

```diff
@@ -182,7 +212,7 @@ export function findSet(deck, gameMode = "normal", old) {
 }
 
 export function splitDeck(deck, gameMode = "normal", minBoardSize = 12, old) {
-  let len = Math.min(deck.length, minBoardSize);
+  let len = Math.min(deck.length, modes[gameMode].minSize);
```

Comment:
I think that in terms of balance, 9 cards work better, but this certainly isn't necessary.  I'll look into the logic and see if there's a way to make it work, otherwise we can got with 12.  
As a side note, if we want to have 5-feature set, variable size boards will need to be implemented at some point.  While it probably would work to use 12 cards and just use the normal setcheck to extend the board, I feel like it would be jarring for the board to be 15 cards most of the time and both grow and shrink beyond this

Reply:
There is definitely a way to make it work, but to be frank, you need to convince me that your code isn't broken like it is right now. It's really important that the site is stable; we have millions of games in the database, and restoring it from a catastrophic issue is not something I want to do.

Reply:
Apologies for the bluntness, but as it is right now, the code footprint & changes are too large. If I hadn't been slightly paranoid and looked very carefully at this file, the bug would probably have leaked in and required hours of production debugging to triage and fix, and possibly a restore of the database from a backup.